lib/rules/web/admin_console/4.4/deployment.xyaml

```diff
@@ -61,3 +61,39 @@ check_sections_on_container_details:
       xpath: //h2[contains(@class,'co-section-heading') and .='Image Details']
   - selector:
       xpath: //h2[contains(@class,'co-section-heading') and .='Network']
+check_hpa_metrics_table:
+  elements:
+  - selector:
+      xpath: //div/h2[contains(@data-test-section-heading,'Metrics')]/following-sibling::div[contains(.,'Type')]/../div[contains(.,'Current')]/../div[contains(.,'Target')]
```

Comment:
This looks a little complicated, how about this 
```
elements:
- selector:
     xpath: //h2[contains(@data-test-section-heading,'Metrics')]/..//div[contains(@class,'co-m-table-grid__head')]/div[contains(.,'Type')]
- selector:
     xpath: //h2[contains(@data-test-section-heading,'Metrics')]/..//div[contains(@class,'co-m-table-grid__head')]/div[contains(.,'Current')]
- selector:
     xpath: //h2[contains(@data-test-section-heading,'Metrics')]/..//div[contains(@class,'co-m-table-grid__head')]/div[contains(.,'Target')]

Reply:
@yapei @SSshahan  Did you got agreement on this ?

Reply:
@liangxia  yes, we have come to agreement on this